near-sdk/src/types/primitives.rs

```diff
@@ -0,0 +1,45 @@
+use near_primitives_core::hash::CryptoHash;
+
+pub use near_primitives_core::runtime::fees::RuntimeFeesConfig;
```

Comment:
This should be restricted to non-wasm32 as well, right?

Reply:
yeah, was going to do in follow up PR, but can make that change now (was being cautious to avoid removing if needed)